msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -148,9 +148,7 @@ The final step is to sideload your updated message extension ([app package](/mic
 
     :::image type="content" source="images/toolkit-zip-teams-metadata-package.png" alt-text="The screenshot is an example that shows 'Zip Teams metadata package' option in Teams Toolkit extension for Visual Studio Code.":::
 
-1. Sign in to Teams with your sandbox tenant account, and toggle into  *Developer Preview* mode. Select the ellipsis (**...**) menu by your user profile, then select: **About** > **Developer preview**.
```

Comment:
@erikadoyle - pretty sure we want to remove this given https://github.com/MicrosoftDocs/msteams-docs/pull/7930 right?